dropshot/tests/common/mod.rs

```diff
@@ -29,15 +29,18 @@ pub fn test_setup(
      */
     let config_dropshot: ConfigDropshot = Default::default();
 
-    let config_logging = ConfigLogging::File {
+    let logctx = create_log_context(test_name);
+    let log = logctx.log.new(o!());
+    TestContext::new(api, 0 as usize, &config_dropshot, Some(logctx), log)
+}
+
+pub fn create_log_context(test_name: &str) -> LogContext {
+    let log_config = ConfigLogging::File {
         level: ConfigLoggingLevel::Debug,
         path: "UNUSED".to_string(),
-        if_exists: ConfigLoggingIfExists::Fail,
+        if_exists: ConfigLoggingIfExists::Append,
```

Comment:
Why this change?  Isn't the name supposed to be unique?

Reply:
Accidental, will change. I had copied that block from a different test, which was using Append, and didn't notice the difference